src/renderer/src/components/browser-ui/browser-sidebar/_components/pin-grid/pinned-tab-button.tsx

```
@@ -48,32 +83,166 @@ export function PinnedTabButton({ faviconUrl, isActive }: { faviconUrl: string;
+        style={activeBorderStyle}
+        onClick={handleClick}
+        onDoubleClick={onDoubleClick}
+        onContextMenu={handleContextMenu}
+      >
+        <div className={cn("size-full", isActive && "bg-white/80 dark:bg-white/30")}>
+          <div className={cn("size-full", "flex items-center justify-center")} style={activeOverlayStyle}>
+            <div className="relative size-5">
+              <img
+                src={faviconUrl || undefined}
+                className="absolute rounded-sm user-drag-none object-contain overflow-hidden"
+              />
```

Biome (2.4.6)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

(lint/a11y/useAltText)